packages/extractor/package.json

```diff
@@ -34,37 +34,41 @@
     "@oclif/config": "^1.17.0",
     "@oclif/plugin-help": "^3.2.0",
     "@types/debug": "^4.1.5",
-    "@types/lodash": "^4.14.149",
+    "@types/lodash": "^4.14.165",
     "chaindown": "^0.1.0-next.1",
-    "debug": "^4.1.1",
+    "debug": "^4.3.1",
     "dedent": "^0.7.0",
     "doctoc": "^1.4.0",
-    "fs-jetpack": "^2.2.3",
-    "globby": "^11.0.0",
+    "fs-jetpack": "^4.1.0",
+    "globby": "^11.0.1",
     "json5": "^2.1.3",
     "kleur": "^4.1.3",
-    "lodash": "^4.17.15",
-    "prettier": "^2.1.2",
-    "ts-morph": "^7.0.0",
+    "lodash": "^4.17.20",
+    "prettier": "^2.2.1",
+    "ts-morph": "^9.1.0",
     "typescript": "^4.1.2"
   },
   "devDependencies": {
-    "@oclif/dev-cli": "1.22.2",
+    "@oclif/dev-cli": "1.24.1",
     "@prisma-labs/prettier-config": "0.1.0",
+    "@types/decompress": "^4",
     "@types/dedent": "^0.7.0",
-    "@types/jest": "25.1.3",
+    "@types/jest": "26.0.15",
     "@types/json5": "^0.0.30",
-    "@types/node": "13.7.6",
-    "@types/prettier": "1.19.0",
+    "@types/node": "14.14.10",
+    "@types/prettier": "2.1.5",
+    "axios": "^0.21.0",
```

Comment:
Used to be my goto as well. But what about https://github.com/sindresorhus/got?

Reply:
Will have a look. That guy is a god